apps/staking/components/Form/StakeAmountField.tsx

```diff
@@ -116,18 +115,18 @@ export default function StakeAmountField({
               <ActionModuleTooltip>
                 {actionModuleSharedDictionary('stakeAmountDescription')}
               </ActionModuleTooltip>
-              {watchedStakeAmount &&
-              tokenBalance &&
-              tokenBalance < stringToBigInt(watchedStakeAmount, SENT_DECIMALS) ? (
-                <AlertTooltip
-                  tooltipContent={dictionary('notEnoughTokensAlert', {
-                    walletAmount: formatSENTBigInt(tokenBalance),
-                    tokenAmount: formatSENTBigInt(
-                      stringToBigInt(watchedStakeAmount, SENT_DECIMALS)
-                    ),
-                  })}
-                />
-              ) : null}
+              {/*{watchedStakeAmount &&*/}
```

Comment:
WIP? Should you uncomment it now?

Reply:
Staking and Registration are being rewritten, so these comments out are to keep the compiler happy, nothing works anymore.